lib/Constants.js

```diff
@@ -35,6 +35,8 @@ module.exports = {
 		'Key1'          : 1 << 26,
 		'Key3'          : 1 << 27,
 		'Key2'          : 1 << 28,
+		'Score'         : 1 << 29,
```

Comment:
```suggestion
		'ScoreV2'       : 1 << 29,
```
As in [the docs](https://github.com/ppy/osu-api/wiki#mods).